### PR TITLE
Switch to PELUX distro

### PR DESCRIPTION
**Pull request by GordanM**
The PELUX distribution is a derived poky distribution.

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

### CODE REVIEW

File: conf/local.conf.sample [hunk 1]
```diff
@@ -9,9 +9,6 @@ BB_NUMBER_THREADS ?= "${@oe.utils.cpu_count()}"
 PARALLEL_MAKE ?= "-j ${@oe.utils.cpu_count()}"
 
 PACKAGE_CLASSES ?= "package_rpm"
-EXTRA_IMAGE_FEATURES_append = " debug-tweaks "
```
Comment (rpannek) [inline conf/local.conf.sample:12]: So where did those go, into the image? I always like some comment on this in the commit message so it's clear if this got just removed and for what reason and if just moved where to.

Comment (GordanM) [reply conf/local.conf.sample:12]: @rpannek valid point.
The IMAGE definitions will be found in the image recipe (https://github.com/Pelagicore/meta-pelux/pull/13/commits/5f87c0f37a89d80575f0708da4df5e0d4c9d4ee1#diff-b6bafc97f66a201ce881d55c0b4998fcR27), while the DISTRO definitions are found in the distro .conf file.

Comment (DunderRoffe) [reply conf/local.conf.sample:12]: Could you add that to the commit message? :)
That would also be good for the intel PR

**Review by DunderRoffe — COMMENTED** [review]
Looks good.  However, as stated above it would be nice with some comments in the commit message

**Review by rpannek — APPROVED** [review]

**Pull request merged**